src/main/java/javaClasses/homework_5/Heghine_Khachatryan/interface_part/CruiseShip.java

```diff
@@ -0,0 +1,22 @@
+package javaClasses.homework_5.Heghine_Khachatryan.interface_part;
+
+public class CruiseShip implements Bookable, Stoppable {
+    private String transportName;
+
+    public CruiseShip(String transportName) {
+        this.transportName = transportName;
+    }
+
+    public String getTransportName() {
```

Comment:
Please add @Override annotations over the overriden methods. Of course it will work in this way too, but it can be useful habit to use that annotation